README.md

```diff
@@ -222,6 +222,136 @@ Descope SDK is automatically refreshes the session token when it is about to exp
 If the Descope project settings are configured to manage tokens in cookies.
 you must also configure a custom domain, and set it as the `baseUrl` to the `descope` plugin. See the above [`plugin` usage](#add-descope-plugin-to-your-application) for usage example.
 
+### Widgets
+
+Widgets are components that allow you to expose management features for tenant-based implementation. In certain scenarios, your customers may require the capability to perform managerial actions independently, alleviating the necessity to contact you. Widgets serve as a feature enabling you to delegate these capabilities to your customers in a modular manner.
+
+Important Note:
+
+- For the user to be able to use the widget, they need to be assigned the `Tenant Admin` Role.
+
+#### User Management
+
+The `User Management` widget will let you embed a user table in your site to view and take action.
+
+The widget lets you:
+
+- Create a new user
+- Edit an existing user
+- Activate / disable an existing user
+- Reset an existing user's password
+- Remove an existing user's passkey
+- Delete an existing user
+
+Note:
+
+- Custom fields also appear in the table.
+
+###### Usage
+
+```vue
+<template>
+	<div class="manage-users-wrapper">
+		<h1>Manage Users</h1>
+		<UserManagement :tenant="tenant" />
+	</div>
+</template>
+
+<script setup>
+import { UserManagement } from '@descope/vue-sdk';
+
+const tenant = 'tenantId';
+</script>
+
+<style>
```

Comment:
Don't think it is needed

Reply:
its in the example like this

Reply:
still, it should not be like that in readme.